### PR TITLE
Adjust glue xml file path handling

### PR DESCRIPTION
**Pull request by QuantumChemist**
Made the glue xml file path handling more flexible

### CODE REVIEW

File: tests/fitting/test_fitting_flows.py [hunk 1]
```diff
@@ -361,9 +361,9 @@ def test_mlip_fit_maker_glue_xml(
     # Test to check if gap fit runs with default hyperparameter sets (i.e. include_two_body and include_soap is True)
     gapfit = MLIPFitMaker(
         mlip_type="GAP",
+        glue_file_path=glue_file
     ).make(
         species_list=["Si"],
-        isolated_atoms_energy=[-0.82067307],
```
Comment (JaGeo) [inline tests/fitting/test_fitting_flows.py:366]: Why removed?

**Comment by JaGeo** [comment]
Just merge if you think it is ready but maybe also add another test for a different name!

File: tests/fitting/test_fitting_flows.py (hunk 1 above)
Comment (QuantumChemist) [reply tests/fitting/test_fitting_flows.py:366]: First reason is that the data type changed to dict and second is that for GAP we retrieve the energy from the isolated atom energy calculation. This keyword is needed for the other MLIP architectures.

**Comment by QuantumChemist** [comment]
> Just merge if you think it is ready but maybe also add another test for a different name!

Ok. I can add a test for another name. 👍🏻 
The merging is blocked as it requires a review.

**Comment by QuantumChemist** [comment]
> pytest tests/fitting/test_fitting_flows.py --durations-path ./tests/test_data/.pytest-split-durations --store-durations

I have a problem to do that
```
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --durations-path ./tests/test_data/.pytest-split-durations --store-durations
  inifile: /home/certural/git/autoplex/pyproject.toml
  rootdir: /home/certural/git/autoplex

```

**Comment by naik-aakash** [comment]
> > pytest tests/fitting/test_fitting_flows.py --durations-path ./tests/test_data/.pytest-split-durations --store-durations
> 
> I have a problem to do that
> 
> ```
> ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
> pytest: error: unrecognized arguments: --durations-path ./tests/test_data/.pytest-split-durations --store-durations
>   inifile: /home/certural/git/autoplex/pyproject.toml
>   rootdir: /home/certural/git/autoplex
> ```

Do you have pytest-split installed in your env?

**Review by naik-aakash — APPROVED** [review]

**Comment by QuantumChemist** [comment]
I forgot to push the duration file, I will do it in a new PR

**Pull request merged**